components/mbedtls_cxx/mbedtls_wrap.cpp

```diff
@@ -27,11 +40,30 @@ bool Tls::init(is_server server, do_verify verify)
     if (verify == do_verify{true}) {
         mbedtls_ssl_conf_ca_chain(&conf_, &ca_cert_, nullptr);
     }
+
+    if (is_server_ && is_dtls_) {
+        if (!init_dtls_cookies()) {
+            return false;
+        }
+    }
+
     ret = mbedtls_ssl_setup(&ssl_, &conf_);
     if (ret) {
         print_error("mbedtls_ssl_setup", ret);
         return false;
     }
+
+    if (timeout) {
+        mbedtls_ssl_set_timer_cb(&ssl_, &timer_, mbedtls_timing_set_delay, mbedtls_timing_get_delay);
+    }
+
+    if (is_server_ && is_dtls_ && client_id != const_buf{}) {
+        client_id_ = client_id;
+        if (!set_client_id()) {
+            return false;
```

Comment:
Missing. error log

Reply:
will add

Reply:
the error log has been added to the `set_client_id()` method.